Dockerfile

```diff
@@ -15,7 +15,7 @@ USER root
 # install all deps required to build modules
 RUN apt-get update \
     && DEBIAN_FRONTEND=noninteractive \
-    apt-get install --assume-yes \
+    apt-get install --no-install-recommends --assume-yes \
```

Comment:
https://askubuntu.com/questions/65081/what-are-the-implications-of-no-install-recommends-apt-get-install